package.json

```diff
@@ -7,14 +7,18 @@
     "node": ">=8.10.0"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha --reporter spec",
+    "docs": "./node_modules/.bin/jsdoc --configure jsdoc.json",
     "lint": "./node_modules/.bin/eslint **/*.js --ignore-pattern node_modules/",
-    "docs": "./node_modules/.bin/jsdoc --configure jsdoc.json"
+    "lint:watch": "chokidar --initial './lib' './test' './examples' -c 'yarn lint'",
```

Comment:
do we need `./node_modules/.bin/chokidar` ?

Reply:
Here we're technically using `chokidar-cli`, which already implicitly pulls in `chokidar` as a dependency.

Reply:
I think the pattern we want to follow is for libraries from `package.json` it is safer to reference any cli/bins as `./node_modules/.bin/xxx`.

Reply:
(yarn would be an exception, as it's recommended to always be installed globally)

Reply:
Updated to use `./node_modules/.bin`